packages/oid4vci-holder/src/machine/oid4vciMachine.ts

```diff
@@ -245,6 +261,10 @@ const createOID4VCIMachine = (opts?: CreateOID4VCIMachineOpts): OID4VCIStateMach
       [OID4VCIMachineStates.transitionFromSetup]: {
         id: OID4VCIMachineStates.transitionFromSetup,
         always: [
+          {
+            target: OID4VCIMachineStates.getFederationTrust,
+            cond: OID4VCIMachineGuards.oid4vciIsOIDFOriginGuard && OID4VCIMachineGuards.oid4vciHasTrustedAnchors,
```

Comment:
```
 cond: [OID4VCIMachineGuards.oid4vciIsOIDFOriginGuard , OID4VCIMachineGuards.oid4vciHasTrustedAnchors]
 ```
 Now you are &&ing two strings

Reply:
which seem to work

Reply:
I don't see why
```
console.log('alice' && 'bob'); // Outputs: 'bob'
console.log('' && 'bob');      // Outputs: ''
console.log('alice' && '');    // Outputs: ''
```

Reply:
as discussed, it was only evaluating the last one. so for now i just combined the guards into one. one guard was not really doing anything yet anyways